### DISCUSSION TITLE
how to findOne by min date?

### DISCUSSION THREAD

**Discussion by crapthings** [ujjwalguptaofficial/JsStore#240] [Q&A] [answered]
```js
db.select({ from: 'Records', order: { by: 'date', type: 'asc' }, limit: 1 }).then(console.log)
```

this doesn't work

**Comment by ujjwalguptaofficial** [comment, marked as answer]
you can use aggregate for finding minimum value of a column, check this example - https://ujjwalguptaofficial.github.io/idbstudio/?db=Demo&query=select(%7B%0A%20%20%20%20from%3A%20'Orders'%2C%0A%20%20%20%20aggregate%3A%20%7B%0A%20%20%20%20%20%20%20%20min%3A%20'orderDate'%0A%20%20%20%20%7D%0A%7D)

here is aggregate docs - https://jsstore.net/tutorial/select/aggregate/
